packages/farmhash/farmhash.0.4/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "github@ahrefs.com"
+authors: "Ahrefs <github@ahrefs.com>"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ahrefs/ocaml-farmhash"
+dev-repo: "git+https://github.com/ahrefs/ocaml-farmhash.git"
+bug-reports: "https://github.com/ahrefs/ocaml-farmhash/issues"
+tags: ["org:ahrefs" "clib:farmhash"]
+build: [ "dune" "build" "-p" name "-j" jobs ]
+depends: [
+  "ocaml" {>= "4.01.0"}
+  "dune" {>= "2.0"}
+]
+depexts: [
```

Comment:
Can you remove the depexts and instead use `conf-c++`, which includes the depexts for a lot more systems? I would have done it myself, but I am not allowed to edit this PR